scene/gui/dialogs.cpp

```diff
@@ -39,7 +39,7 @@
 
 void AcceptDialog::_input_from_window(const Ref<InputEvent> &p_event) {
 	Ref<InputEventKey> key = p_event;
-	if (close_on_escape && key.is_valid() && key->is_pressed() && key->get_keycode() == Key::ESCAPE) {
+	if (close_on_escape && key.is_valid() && key->is_pressed() && key->is_action_pressed(SNAME("ui_cancel", false, true))) {
```

Comment:
This is still unnecessary, `is_pressed` isn't needed if you're going to use `is_action_pressed`

Also you need to write `SNAME("ui_cancel")`

Reply:
Got it I missed that is_pressed isn't necessary on the original review.

Also I see I messed up my parenthesis there, fixing it now